common/actions.js

```diff
@@ -98,7 +98,15 @@ export const getSlateById = async (data) => {
 
 export const getSlatesByIds = async (data) => {
   await Websockets.checkWebsocket();
-  return await returnJSON(`/api/slates/get`, {
+  return await returnJSON(`/api/v1/explore`, {
+    ...DEFAULT_OPTIONS,
+    body: JSON.stringify({ data }),
+  });
+};
+
+export const getSocial = async (data) => {
+  await Websockets.checkWebsocket();
```

Comment:
Why do we need to checkWebsocket here? (cc: @martinalong)

Reply:
actually I'm just following the format here. I could remove it.
but when do we need to checkWebsocket then?

Reply:
We use WebSockets to deliver real time updates back to the browser really fast. We check if the WebSocket is alive so when we perform this action, we know we will receive a socket update instead of having to manually update the state of the react application.

I'm assuming these WebSocket checks ensure that we're still connected/reconnect, you might not need it for social!

Reply:
Yeah exactly what jim said. You need it if the endpoint it calls has something like "hydratePartialSlates" or "hydratePartialLibrary". That means it uses websockets to update the UI, and you want to make sure it checks (and if needed, reconnects) websockets before it makes the call. In this case, not necessary since the info just comes back in the response

Reply:
If you take a look at the other calls, you'll see that some of the functions in action.js have checkWebsocket and some don't. I left a note at the top explaining it